README.md

```diff
@@ -1,51 +1,31 @@
 
-**NOTE:**  The above installation instructions currently only work on Ubuntu 16.04 (xenial) and 18.04 (bionic). Please be aware that the script will install the latest *development preview* of Unity8 for desktop.  ***Expect bugs!***  Only install this preview if you understand the risks and are willing/able to fix your system if the preview breaks anything.  You are advised *not* to install this preview onto a production system.
+Before you start, [install Clickable](https://clickable-ut.dev/en/latest/install.html).
 
-See [CODING](CODING) for build instructions.
+Now, clone this repository to your computer: `git clone https://github.com/ubports/unity8.git`
 
+Move into this directory: `cd unity8`
 
-### Where to report issues
+Now you can use the full suite of tools provided by this repository's [clickable.json](clickable.json). For example:
 
-Issues related to the version of Unity8 for desktop (the latest development preview) installed using the above instructions should be reported on the [Unity8 issue tracker](https://github.com/ubports/unity8/issues) in *this* repository.
-
-Note that issues related to the version of Unity8 that currently runs on Ubuntu Touch devices should *NOT* be reported in this repository, but instead should be reported on the [Ubuntu Touch issue tracker](https://github.com/ubports/ubuntu-touch/issues). This will change once Ubuntu Touch moves to the latest version of Unity8.
-
-
-### What works / what does not work
-
-*This list is incomplete, please help improve it*
-
-What works:
-- Most Wayland apps work
-- Many report being able to log in using LightDM, but may require a few attempts
-
-Does not work:
-- Most Xapps don't (yet) work
-- Proprietary drivers don't (yet) work
-- Many report being unable to log in using GDM
-
-
-### Get involved
-
-* [Kanban-Board](https://github.com/ubports/unity8/projects/1)
-* If you are interesting in helping us in any way you can join [Unity8 Forum](https://forums.ubports.com/category/36/unity8).
-* For reporting bugs or filing feature-requests, please use the project’s [issues page](https://github.com/ubports/unity8/issues).
-* For IM you can join our Telegram group [for Developer users](https://t.me/UBports_Unity8).
+* `clickable ide qtcreator` will open QtCreator with this repository open as a project. You can edit and build the project this way, but running the tests will be a bit difficult.
+* `clickable build-libs --dirty --arch=amd64` will build Lomiri in the same environment as it would receive in Ubuntu Touch.
```

Comment:
No need for `--arch=amd64` because the architecture is restricted by the clickable.json `restrict_arch` field. The `--dirty` flag can be removed as well, because `build-libs` does not include cleaning the build dir.

Reply:
Got it.